docroot/themes/humsci/humsci_basic/src/scss/README.md

```diff
@@ -0,0 +1,14 @@
+# Theming
+
+The Humsci Basic theme provides the styles for a variety of different themes.
+
+## Adding a color palette for a theme
+1. Create a new settings partial with color pairings and globals SCSS maps
```

Comment:
maybe write at the end of this, for example `_variables.new-theme-pairings.scss`

Reply:
I think examples are really helpful if you wanted to use colorful ocean as that example in here